descriptions.go

```diff
@@ -186,6 +192,8 @@ type (
 		SessionURL *string                   `json:"sessionUrl,omitempty"`
 		URL        *TranslatedString         `json:"url,omitempty"`
 		InApp      *bool                     `json:"inapp,omitempty"`
+
+		languages []string
```

Comment:
It looks a bit weird to me to add the languages separately to the struct, while the languages should already be present in all `TranslatedString`s. It seems more logical to me to give this `languages` as a parameter to `Validate`, or to add a check in the `Validate` function of `IssueWizard` that all its `IssueWizardItem`s contain all expected languages.

Reply:
We could do that, but it would create an inconsistency: a lot of other structs that contain translatable strings have a `lanuages []string` like this one. Namely, the `SchemeManager`, `Issuer`, `CredentialType`, `RequestorScheme`, `RequestorInfo`, and `IssueWizard` structs. Additionally, none of the other `validate` functions (except the one of `TranslatedString` which is special anyway) take languages as parameter.

As to:
> or to add a check in the `Validate` function of `IssueWizard` that all its `IssueWizardItems` contain all expected languages.

We could do that, but it seems cleaner to me to leave this check to the `validate` function of the `IssueWizardItem` itself.

Reply:
That's a good point. It's both not very ideal. So let's go for consistency then.